fabric-gametest-api-v1/src/main/java/net/fabricmc/fabric/mixin/gametest/TestServerMixin.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.gametest;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import net.minecraft.test.TestServer;
+
+@Mixin(TestServer.class)
+public abstract class TestServerMixin {
+	@Inject(method = "isDedicated", at = @At("HEAD"), cancellable = true)
+	public void isDedicated(CallbackInfoReturnable<Boolean> cir) {
+		cir.setReturnValue(true);
```

Comment:
Why is this needed?

Reply:
I forget as I wrote this months ago :D I will check and get back to you.

Reply:
I think this is used to allow dedicated server commands to be registered. I think when I first wrote this I was running with all the fabric test mods and it would cause the server test to fail. Seems reasonable to leave enabled, I will add a comment.